src/core/room-lifecycle-manager.ts

```diff
@@ -11,9 +11,10 @@ import { DefaultStatus, NewRoomStatus, RoomLifecycle, RoomStatusChange } from '.
  */
 export interface ContributesToRoomLifecycle extends HandlesDiscontinuity {
   /**
-   * Gets the channel on which the feature operates.
+   * Gets the channel on which the feature operates. This promise is never
+   * rejected except in the case where room initialization is cancelled.
```

Comment:
small typo - `canceled`

Reply:
Done, and added cspell config to catch `"cancelled"`